docs/compose/setup.rst

```diff
@@ -31,7 +31,7 @@ Sets the ``TLS_FLAVOR`` to one of the following
 values:
 
 - ``cert`` is the default and requires certificates to be setup manually;
-- ``letsencrypt`` will use the *Letsencrypt!* CA to generate automatic ceriticates;
+- ``letsencrypt`` will use the *Letsencrypt!* CA to generate automatic certicates;
```

Comment:
uhhm… isn't this still a typo?

shouldn't it be:
```suggestion
- ``letsencrypt`` will use the *Letsencrypt!* CA to generate automatic certificates;
```

Reply:
omfg, now i really feel ashamed :sweat_drops:

Reply:
o.o How this happened!! I was totally blind!